.travis.yml

```diff
@@ -81,8 +81,8 @@ matrix:
           env: PYTHON_VERSION=2.7 SETUP_CMD='test -a "--mpl"'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
                if [[ $PYTHON_VERSION == 2.7 ]];
-               then PIP_DEPENDENCIES='jplephem mock';
-               else PIP_DEPENDENCIES='jplephem';
+               then PIP_DEPENDENCIES='jplephem mock pytest-mpl';
+               else PIP_DEPENDENCIES='jplephem pytest-mpl';
```

Comment:
Doesn't matter to much, but you can add this to the conda dependencies as we build it in the astropy channel. Also I don't get the conditionals here, when does it go to the else part?